src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java

```diff
@@ -168,6 +168,18 @@
     )
     private YesOrNo continueApplication;
 
+    @CCD(
+        label = "Is the information in this application still correct?"
+    )
+    private YesOrNo confirmInformationStillCorrect;
```

Comment:
I would put these questions in the section they come up in, like ConditionalOrder or something

Reply:
Is it not a similar situation to the one the other day where both applicants will be answering the question or is it just an app1 question @linusnorton ?